components/image-grid/image-grid.js

```diff
@@ -1,8 +1,22 @@
 import styles from "./image-grid.module.scss";
 import ArrowLink from "../arrow-link/arrow-link";
 
+import { useEffect, useRef, useMemo, createRef } from "react";
+import {scrollListener} from '../../utils/animations'
+import { RefreshSharp } from "@mui/icons-material";
+
 const ImageGrid = (props) => {
-  const sections = props.items.map((item) => {
+
+  const ref = useRef([]);
```

Comment:
When you initialize it with an array does it just add each new ref to that array?